.circleci/config.yml

```diff
@@ -126,10 +126,34 @@ references:
           owner=${CIRCLE_PROJECT_USERNAME} repo=${CIRCLE_PROJECT_REPONAME} \
           tag=${CIRCLE_TAG} ASSETS=${PACKAGES_DIR:?}/*
 
+  fail_notification: &fail_notification
+    run:
+      name: Send failure notification
+      environment:
+        NOTIFY_BRANCH: PT-156197975_failing_master_notifications
```

Comment:
?

Reply:
testing ATM

Reply:
Only this needed to be changed then looks good.

Reply:
done in 9b407b7